src/datreant/core/agglimbs.py

```diff
@@ -52,14 +51,14 @@ def __str__(self):
         return out
 
     @staticmethod
-    def _setter(self, val):
+    def _setter(other, val):
```

Comment:
@dotsdl did you want this to be `self` or an `other` that is changed by the method?